src/assets/usage-examples-code/graphics/fill_rectangle/fill_rectangle-1-rectangle-color.txt

```diff
@@ -0,0 +1,3 @@
+### Rectangle Color
+
+The following code shows examples of using [Fill Rectangle](/api/graphics#fill-Rectangle) to show how to draw a blue rectangle.
```

Comment:
Change 'blue' to 'blue-colored filled' to be more specific about the color used.

Reply:
Or it could be "... to show how to draw a filled-in blue rectangle."